tests/sqlparser_mssql.rs

```diff
@@ -495,6 +497,8 @@ fn parse_convert() {
         ParserError::ParserError("Expected: an expression, found: )".to_owned()),
         ms().parse_sql_statements(error_sql).unwrap_err()
     );
+
+    ms().verified_expr("TRY_CONVERT(VARCHAR(MAX), 'foo')");
```

Comment:
with the dialect method we can move this over to common.rs  as a generic test over the dialects that support the feature